src/route/nodes/netid/nodeEdit/NodeEdit.tsx

```diff
@@ -28,6 +47,81 @@ export const NodeEdit: React.FC<{
   const node = useNodeById(decodeURIComponent(nodeId))
   const network = useNetwork(netid)
 
+  const createipValidation = useMemo(
```

Comment:
createipValidation should be camel cased properly -> createIPValidation or createIpValidation